src/frontends/tensorflow_common/include/helper_ops/string_constant.hpp

```diff
@@ -0,0 +1,59 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include "internal_operation.hpp"
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+
+/// Pseudo-entity for storing strings
+class StringConstant : public InternalOperation {
+public:
+    OPENVINO_OP("StringConstant", "ov::frontend::tensorflow::util", InternalOperation);
+
+    StringConstant(ov::Any data, const std::shared_ptr<DecoderBase>& decoder = std::make_shared<DecoderFake>())
+        : InternalOperation(decoder, {}, 1),
+          m_data(data),
+          m_str() {
+        validate_and_infer_types();
+    }
+
+    StringConstant(std::string& str, const std::shared_ptr<DecoderBase>& decoder = std::make_shared<DecoderFake>())
+        : InternalOperation(decoder, {}, 1),
+          m_data(),
+          m_str(str) {
+        validate_and_infer_types();
+    }
+
+    StringConstant(const std::shared_ptr<DecoderBase>& decoder = std::make_shared<DecoderFake>())
+        : InternalOperation(decoder, {}, 1) {
+        validate_and_infer_types();
+    }
+
+    void validate_and_infer_types() override {
+        set_output_type(0, ov::element::undefined, ov::PartialShape::dynamic());
+    }
+
+    ov::Any get_data() {
+        return m_data;
+    }
+
+    std::string& get_string() {
+        return m_str;
+    }
+
+private:
+    ov::Any m_data;
+    std::string m_str;
```

Comment:
please add shape member:

```
ov::Shape m_shape;
```

Reply:
And what should I put there?

Reply:
shape of string tensor

Reply:
["asd", "asd123", "q"] - [3, ?]? [3]?
"asd" - 1?

Reply:
["asd", "asd123", "q"] - [3]
"asd" - [] (because this is a scalar)